internal/venice-common/src/main/java/com/linkedin/venice/ConfigKeys.java

```diff
@@ -1739,6 +1739,14 @@ private ConfigKeys() {
   // Config to control how much percentage of DVC replica instances are allowed to be offline before failing VPJ push.
   public static final String DAVINCI_PUSH_STATUS_SCAN_MAX_OFFLINE_INSTANCE_RATIO =
       "davinci.push.status.scan.max.offline.instance.ratio";
+  // Denotes whether DVC is allowed to bootstrap via a peer-to-peer blob transfer
+  public static final String DAVINCI_P2P_BLOB_TRANSFER_ENABLED = "davinci.p2p.blob.transfer.enabled";
```

Comment:
Now I am thinking we don't need this config. We've added `blobTransferEnabled` to the version config, and it has implied that this config would apply to the store-version only. 
For server p2p, it should use a cluster-wide config since it doesn't much sense to me that we only enable server p2p to a few stores. @sebas-inf  @sixpluszero

Reply:
would we want a "davinci_ambry_blob_transfer_enabled" as a feature flag in future? if so, maybe we should consider keeping this.